src/Request/PaymentStatusRequest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace OpMerchantServices\SDK\Request;
+
+use OpMerchantServices\SDK\Interfaces\RequestInterface;
+use Respect\Validation\Validator as v;
+use Respect\Validation\Exceptions\NestedValidationException;
+
+/**
+ * Class PaymentStatusRequest
+ *
+ * @package OpMerchantServices\SDK\Request
+ */
+class PaymentStatusRequest implements RequestInterface
+{
+
+    /**
+     * Payment transaction id.
+     *
+     * @var string
+     */
+    protected $transactionId;
+
+    /**
+     * Validates with Respect\Validation library and throws an exception for invalid objects
+     *
+     * @throws NestedValidationException Thrown when the validate() fails.
+     */
+    public function validate()
+    {
+        v::notEmpty()->stringType()->validate($this->transactionId);
```

Comment:
We are in the process of removing the validation library, as it causes some problems with hosting providers that don't have mbstring enabled. 

Could you please check for the empty string with empty() and throw an OpMerchantServices\SDK\Exception\ValidationException in the case that the string is empty.

Otherwise looks all good to me.

Reply:
Done. I added a new commit to this pull request.